lib/tidewave/tools/execute_sql_query.rb

```diff
@@ -19,7 +19,7 @@ class Tidewave::Tools::ExecuteSqlQuery < Tidewave::Tools::Base
 
   arguments do
     required(:query).filled(:string).description("The SQL query to execute. For PostgreSQL, use $1, $2 placeholders. For MySQL, use ? placeholders.")
-    optional(:arguments).value(:array).description("The arguments to pass to the query. The query must contain corresponding parameter placeholders.")
+    optional(:arguments).array(:str?, :bool?, :float?, :nil?, :hash?).description("The arguments to pass to the query. The query must contain corresponding parameter placeholders.")
```

Comment:
Can it also be a nested array? I am not sure if this works:

```suggestion
    optional(:arguments).array(:array?, :str?, :bool?, :float?, :nil?, :hash?).description("The arguments to pass to the query. The query must contain corresponding parameter placeholders.")
```

Reply:
It does !